FlowCrypt/Controllers/Msg/MessageViewController.swift

```diff
@@ -148,26 +148,38 @@ final class MessageViewController: TableNodeViewController {
 }
 
 // MARK: - Message
-
 extension MessageViewController {
     private func fetchDecryptAndRenderMsg() {
         handleFetchProgress(state: .fetch)
 
-        Promise { [weak self] in
-            guard let self = self else { return }
-            let promise = self.messageService.getAndProcessMessage(
-                with: self.input.objMessage,
-                folder: self.input.path,
-                progressHandler: self.handleFetchProgress
-            )
-            let message = try awaitPromise(promise)
-            self.processedMessage = message
-        }
-        .then(on: .main) { [weak self] in
-            self?.handleReceivedMessage()
+        Task {
+            do {
+                processedMessage = try await serviceActor.fetchDecryptAndRenderMsg(message: input.objMessage,
+                                                                                   path: input.path,
+                                                                                   progressHandler: { [weak self] state in
+                                                                                        self?.handleFetchProgress(state: state)
+                                                                                   })
```

Comment:
A detail - I'd prefer formatting like:
```swift
processedMessage = try await serviceActor.fetchDecryptAndRenderMsg(
    message: input.objMessage,
    path: input.path,
    progressHandler: { [weak self] state in
        self?.handleFetchProgress(state: state)
    }
)
```
Else all of the inputs have no space due to large indentation.

Reply:
Okay, I'll use such formatting in the next commits